Documentation/contributing/development/reviewers_committers/review_vendor.rst

```diff
@@ -0,0 +1,117 @@
+.. _cilium_slash_cilium_reviews: https://github.com/cilium/cilium/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _cilium_slash_hubble_reviews: https://github.com/cilium/hubble/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _cilium_slash_cilium_cli_reviews: https://github.com/cilium/cilium-cli/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _cilium_slash_certgen_reviews: https://github.com/cilium/certgen/pulls?q=is%3Apr+is%3Aopen+draft%3Afalse+team-review-requested%3Acilium%2Fvendor
+.. _ladder: https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md
+
+Reviewing Pull Requests
+=======================
+
+This section describes some of the process and expectations for reviewing PRs
+on behalf of @cilium/vendor. Note that :ref:`the generic PR review
+process for Committers <review_process>` still applies, even though it is not
+specific to dependencies.
+
+Existing Dependencies
+---------------------
+
+Updates to existing dependencies most commonly occur through PRs opened by
+`Renovate <renovate_>`_, which is a 3rd party service used throughout the
+Cilium organization. Renovate continually checks repositories for out-of-date
+dependencies and opens new PRs to update any it finds.
+
+.. note::
+   Renovate is configured via a file named ``renovate.json5`` found within the
+   ``.github/`` folder of each repository it is enabled on. Cilium's Renovate
+   configuration is long and complex, having been developed to enable
+   dependency updates to Docker images and GitHub Actions. Only a few pieces
+   of the configuration are relevant to the @cilium/vendor team.
+
+.. _renovate: https://docs.renovatebot.com
+
+When reviewing PRs that update an existing dependency, members of the
+@cilium/vendor team are required to ensure that the update does not include
+any breaking changes or licensing issues. These checks are facilitated via
+GitHub Action CI workflows, which are triggered by commenting ``/test`` within
+a PR. See :ref:`CI  / GitHub Actions <ci_gha>` for more information on their
+use.
+
+New Dependencies
+----------------
+
+When a new dependency is added as part of a PR, the @cilium/vendor team will
+be assigned to ensure the new dependency meets the following criteria:
+
+1. The new dependency must add functionality that is not already provided within
+   an existing dependency, an internal package to the project, or Go's
+   standard library.
+2. The functionality provided by the new dependency must be non-trivial to
+   re-implement manually.
+3. The new dependency must be actively maintained, having new commits and/or
+   releases within the past year.
+4. The new dependency must appear to be of generally good quality.
+
+These criteria ensure the long-term success of the project by justifying the
+inclusion of the new dependency into the project's codebase.
+
```

Comment:
What about some exceptions? Should we document cases in which we can allow some exceptions from the above rules?

Reply:
I'm not sure exceptions would be appropriate here for these guidelines. Do you have an example?

Reply:
I'm thinking about 3 and 4, I can see cases in which we would like to experiment using some dependencies that are not fitting in these criteria, but as you mentioned above, I think we can discuss this on each PR or during that meeting.

Reply:
If its' not already clear, we should make it clear that this guide provides a set of guidelines for members of @cilium/vendor. Best practices, not strict rules. We are trusting @cilium/vendor to make good decisions about depnedencies. If we think these guidelines should provide sufficient framework to evaluate dependencies, then they're doing their job. We can discuss cases where a dependency diverges from these guidelines and ultimately it'll come down to a tradeoff of the benefits and the sustainability of the dependency.

So far, we haven't had any written guidelines. The docs added in this PR provide a huge step forward. The best way to evaluate how well these guidelines work is to put them in place and start to test them. If we find that the guidelines do not provide good enough guidance, we can iterate further. If we find that many dependencies we change/add/update require exceptions, then we should learn from that and either adapt the guidelines or establish an exceptions process. I don't think we need to front-load all of that right now. I'd rather get *some* guidelines into the tree and iterate from there.

Reply:
>That said, unless I see a specific example in front of me I think it's hard for me to think about how these rules will be insufficient to cover any new dependency we consider.

This is where my head is at. @ovidiutirla what would you think about the suggestion @joestringer had here:

> The best way to evaluate how well these guidelines work is to put them in place and start to test them. If we find that the guidelines do not provide good enough guidance, we can iterate further.

Reply:
I'm going to resolve this conversation for now so we can get this PR merged. Let's take the discussion to the community slack.